VERSION

```diff
@@ -1 +1 @@
-3.6.2
+3.7.0
```

Comment:
only nitpicking because there's other changes to be done as well, but:
files should have newlines at the end. I can't remember the reason why... but that's just what we do, and what the github symbol at the end of this line means :P